subsys/bluetooth/host/Kconfig

```diff
@@ -760,6 +760,23 @@ config BT_SCAN_WITH_IDENTITY
 	  for for the local identity. If this use case is required, then enable
 	  this option.
 
+config BT_SCAN_AND_INITIATE_IN_PARALLEL
+	bool "Allow concurrent scanning and initiating"
+	depends on (BT_CENTRAL && BT_OBSERVER)
+	select BT_EXT_ADV if BT_BROADCASTER
+	select BT_SCAN_WITH_IDENTITY if !BT_PRIVACY
+	help
+	  Allow concurrent scanning and initiating.
+	  This will allow the application to initating a connection
+	  to a peer device without stopping the scanner.
+	  If privacy is disabled, the scanner will use its identity
+	  address.
+	  This feature is only available when extended advertising
+	  HCI commands are used to prevent degraded performance
+	  when the advertiser is used.
+	  Time-limited scanning is also not supported when this
```

Comment:
```suggestion
	  Controller-based scanning timeout is not supported when this
```

Reply:
This is a host-specific config. The timeout specified in the host API will always be translated to a controller-based timeout, that is why I left out that detail. Maybe it should be rephrased?

Reply:
what about "scanning with a timeout is not supported when this ..."?